velox/connectors/hive/SplitReader.h

```diff
@@ -137,8 +144,6 @@ class SplitReader {
   std::shared_ptr<io::IoStatistics> ioStats_;
   dwio::common::ReaderOptions baseReaderOpts_;
   dwio::common::RowReaderOptions baseRowReaderOpts_;
-
- private:
```

Comment:
NYC: could you also help move const members first in this class? Shall we also init emptySplit_{false}  Thanks!